src/Raven.Server/Web/Authentication/AdminApiKeysHandler.cs

```diff
@@ -23,7 +23,8 @@ public class AdminApiKeysHandler : AdminRequestHandler
         public async Task Put()
         {
             var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
-
+            if (name.ToLowerInvariant().StartsWith("raven"))
```

Comment:
Use `StartsWith` that does case insensitive search here.
We want to check for `Raven/`.
If they have an API Key called "Ravenous", that is fine, it is just `Raven/` that is reserved.